pkgs/by-name/gd/gdtoolkit_3/package.nix

```diff
@@ -23,15 +22,15 @@ let
   };
 in
 python.pkgs.buildPythonApplication rec {
-  pname = "gdtoolkit";
-  version = "3.3.1";
+  pname = "gdtoolkit_3";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

`The pname attribute should be identical to the upstream package name.`

Reply:
Since both versions of gdtoolkit may be in use, I followed what was chosen for Godot 3 and 4. There may be a better way to do this that I am missing, so please let me know if that is the case!

https://github.com/NixOS/nixpkgs/blob/614b4613980a522ba49f0d194531beddbb7220d3/pkgs/development/tools/godot/3/default.nix#L29
https://github.com/NixOS/nixpkgs/blob/614b4613980a522ba49f0d194531beddbb7220d3/pkgs/development/tools/godot/4/default.nix#L45

Reply:
version will be appended, so you can have both installed `name=godot-3.3.1` and `name=godot-4.4.2`

I am not aware of any issues or reasons for changing `pname`

Reply:
Hmm, I originally thought that as well, but wanted to keep it similar to Godot. I'll check to see if I can find an issue/PR where that was discussed to see why that was chosen.

Reply:
https://github.com/NixOS/nixpkgs/pull/177578

Reply:
It seems the names should be without an `_` then? That's what godot itself is doing, and seems nice to have it consistent.

The docs do indeed say `The pname attribute should be identical to the upstream package name.` (as linked above) but `should` is key here, and this feels like a [valid reason](https://datatracker.ietf.org/doc/html/rfc2119#section-4) for the extra number.